ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -886,6 +886,11 @@ def handle_prompt_template_invoke(self, instance, result, args: List[Any], kwarg
                     break
             chat_template = messages if messages else None
 
+        tags = {}
+        if isinstance(getattr(instance, "metadata", None), dict):
+            metadata = instance.metadata
+            tags = {key: value for key, value in metadata.items() if isinstance(key, str) and isinstance(value, str)}
```

Comment:
we only want str-str key value pairs? or we don't want to add any tags that aren't str-str key value pairs? If the former, can't we just do 
`tags = {str(key): str(value) for key, value in metadata.items()}`
?